### ISSUE TITLE
[Bug Report][3.9.5] v-combobox has duplicate "combobox" roles within the component

### ISSUE THREAD

**Issue by tawago** [vuetifyjs/vuetify#22098]
### Environment
**Vuetify Version:** 3.10.2
**Vue Version:** 3.5.21
**OS:** macOS 10.15.7 (current)

### Steps to reproduce
1. open the reproduction link
2. inspect and search for "combobox"
3. you should see two "combobox"

### Expected Behavior
There should be only one "combobox" role per the component.  Ideally on the `input` element according to https://www.w3.org/WAI/ARIA/apg/patterns/combobox/examples/combobox-autocomplete-both/. 

### Actual Behavior
After 3.9.5, implementation seems to have added "combobox" role on the `input` element however, an ancestor element with classname `v-field ` also has the "combobox" role. Creating issues on a11y users and playwright integrations.

### Reproduction Link
[https://play.vuetifyjs.com/#...](https://play.vuetifyjs.com/#eNpVUslqwzAQ/RWhixxw7S7k0OAEQqA9lR5a6CHJQbHHqUDWqFpCgvG/V7Kz+fbmzfbmSeuWWlPmS62zgwc6o4WDRkvuYLFRhBSHB651D/ugROW4UGDO1JlsdrjD44UiRPIdyPmGri4ZesvNRNhgQ3LNVlyKGo0SnKWErVCi4RVG/BagqHr6HdDsh4pvOHIbwc8JG6H2bHsbvCjym5KrOuWbz/qiYkbadkR03fmwvvX+ssgMhxf5nSEhtKUR2hELzg++iEajcaQlqD7QKwdVSgzUpCO1wYawYCuLdWGBdWNBZB4rk8dJzF/bk2QyX7SDslF5duDSQ2iqsPQNKJf9eTCnL5BQOjRLKRO2NighmHt9E7plk0yC2rvfOLILu4p8OCLIp11KX7Jp9vxEI3jNpjStubSQjr7FwG3/AQ0Ttl4=)

### Images
![https://cdn.cosmicjs.com/2e3e3e70-977e-11f0-bba7-d56988718db7-Screenshot-2025-09-22-at-3-33-14PM.png](https://cdn.cosmicjs.com/2e3e3e70-977e-11f0-bba7-d56988718db7-Screenshot-2025-09-22-at-3-33-14PM.png)

**Comment by J-Sek** [comment]
Caused by [#21903](https://github.com/vuetifyjs/vuetify/pull/21903).
The `role` can be easily fixed, but what about duplicated [aria-*](https://github.com/vuetifyjs/vuetify/blob/master/packages/vuetify/src/components/VMenu/VMenu.tsx#L212-L214), that were supplied to `<input>` by [#22025](https://github.com/vuetifyjs/vuetify/pull/22025)? 🤦🏼‍♂️
